### PR TITLE
Gitlab setup steps

### PR DESCRIPTION
**Pull request by hahnbeelee**

### CODE REVIEW

File: integrations/custom/css.mdx
Comment (rpmccarter) [inline integrations/custom/css.mdx:1]: nice

File: settings/gitlab.mdx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: 'GitLab'
+description: 'Sync your docs with a GitLab repo'
+---
+
+<Check>
+  GitLab is available on the [growth plan](https://mintlify.com/pricing).
```
Comment (rpmccarter) [inline settings/gitlab.mdx]: Can we make this a lil clearer
```suggestion
  GitLab integration is available on the [growth plan](https://mintlify.com/pricing).
```

File: images/gitlab/gitlab-project-id.png
Comment (rpmccarter) [inline images/gitlab/gitlab-project-id.png:1]: There's a lot going on in these images - could you make the red boxes thicker or somehow more pronounced? Could also help if you made your browser window narrower when taking the screenshots

**Review by rpmccarter — APPROVED** [review]
lgtm, images are just a bit difficult to parse

File: images/gitlab/gitlab-project-id.png
Comment (hahnbeelee) [reply images/gitlab/gitlab-project-id.png:1]: will fix them up. ty for the review

**Pull request merged**